taxonomy-editor-frontend/src/pages/editentry/index.jsx

```diff
@@ -19,9 +20,7 @@ const EditEntry = () => {
     const { data: node, error, isPending } = useFetch(url);
     let nodeObject = null;

```

Comment:
should this be an state?, don't quite follow why this is a `let`

Reply:
Since I have to declare `nodeObject` conditionally (based on value of `isPending`), I can't define it using a useState hook. Hence, I'm updating the variable after a conditional statement. Here, it isn't a state variable, but in `AccumulateAllComponents.js` it is.

Reply:
yep, I mean, you could pass it directly instead of creating a `let` , for example `incomingNodeObject={node?.[0]}` . Having the `let` makes it a bit confusing to read and in this case I think it may not be necessary

Reply:
@nobeeakon That's just cool! I haven't heard of the `.?` operator. I'll add it and remove the variable.